src/repositories.ts

```diff
@@ -1,16 +1,14 @@
 import { isLinux } from '@env/platform';
 import type { Uri } from 'vscode';
-import { Schemes } from './constants';
+import { CharCode, Schemes } from './constants';
 import type { RevisionUriData } from './git/gitProvider';
 import { decodeGitLensRevisionUriAuthority } from './git/gitUri';
 import type { Repository } from './git/models/repository';
 import { normalizePath } from './system/path';
 import { UriTrie } from './system/trie';
 import { addVslsPrefixIfNeeded } from './system/vscode/path';
-// TODO@eamodio don't import from string here since it will break the tests because of ESM dependencies
-// import { CharCode } from './string';
 
-const slash = 47; //CharCode.Slash;
+const slash = CharCode.Slash;
```

Comment:
I'd suggest leaving this as `47` -- bundlers don't handle the `const enum` properly and inject the value at the call site, instead they end up with this whole module/object lookup indirection

Reply:
ok. is it the problem of project structure? I mean could we store one file for each enum with default export instead? I know it can be inconvenient now, it's just a clarification

Reply:
replaced